cmd/cluster/core/create.go

```diff
@@ -166,6 +166,7 @@ type RawCreateOptions struct {
 	OLMCatalogPlacement              hyperv1.OLMCatalogPlacement
 	OLMDisableDefaultSources         bool
 	FeatureSet                       string
```

Comment:
we have tests in aws/azure that renders yaml and compares against generated fixtures. We should include a test case there that cover this even thouhg is not platform specific, e.g. see /cluster/azure/create_test.go

Reply:
done